docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
 
-### Entrypoints
+## Usage examples
 
-The `lint` command behaves differently depending on how you pass entrypoints to it and whether the [configuration file](#custom-configuration-file) exists.
+### How to use entrypoints to check all or selected definitions
 
-#### Pass entrypoints directly
+##### Method 1: Pass entrypoints directly
+In this case, `lint` will validate the definitions passed in the command and ignore the `apis` section in `redocly.yaml` or any other customized config file you may be using.
 
 ```bash
-openapi lint openapi/openapi.yaml
+redocly lint openapi/openapi.yaml
 ```
 
-In this case, `lint` will validate the definition(s) passed to the command. The configuration file is ignored.
+The `entrypoints` argument can also use any glob format supported by your file system. For example, `redocly lint ./root-documents/*.yaml`.
 
-The `entrypoints` argument can also use any glob format supported by your file system. For example, `openapi lint ./root-documents/*.yaml`.
+##### Method 2: Pass entrypoints via the config file
 
-#### Pass entrypoints via configuration file
+The `apis` section in the Redocly configuration file (`redocly.yaml`) lets you configure one or more APIs. Each file path is specified (using the `root` property), but files are identified by their name and version in the format `name@version`.
 
-Instead of full paths, you can use names listed in the `apis` section of your Redocly configuration file as entrypoints.
+`@version` is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the `apis` section must be unique.
 
-```bash Command
-openapi lint main
-```
+You can pass `name` or `name@version` in commands like `lint` and `stats` to act only on those APIs. Below is an example `apis` section showing three files identified using `main`, `production` and `sandbox`:
 
 ```yaml Configuration file
 apis:
   main:
-    root: ./openapi/definition.json
+    root: ./openapi.yaml
+    labels:
+      - main
+    lint: []
+    features.openapi: []
+  production:
+    root: ./production.yaml
+    labels:
+      - production
+    lint: []
+    features.openapi: []
+  sandbox:
+    root: ./sandbox.yaml
+    labels:
+      - sandbox
+    lint: []
+    features.openapi: []
 ```
 
-In this case, after resolving the path behind the `main` name (see the `Configuration file` tab), `lint` will validate the `definition.json` file. The presence of the Redocly configuration file is mandatory.
+ In this example, we only want to validate files under `main`:
 
-#### Empty entrypoints
+```bash Command
+redocly lint main
+```
 
-You can omit entrypoints completely when executing the `lint` command.
+##### Method 3: Pass no entrypoints
 
-```bash Command
-openapi lint
+Running `lint` without specifying a particular API definition file checks all definition files in your project directory.
```

Comment:
question: doesn't `lint` in this case validate all APIs listed in the conf file? I don't think it cares about the project directory because you could technically have an API definition located entirely outside of the project directory, but still have it listed in the `apis` section of the conf file. Maybe this needs to be tested?

Reply:
You are correct @skadinna.
```suggestion
Running `lint` without specifying a particular API definition checks all APIs in your configuration file.
```